test/format/checkpoint.c

```diff
@@ -160,8 +160,10 @@ checkpoint(void *arg)
         if (backup_locked)
             lock_writeunlock(session, &g.backup_lock);
 
-        /* Verify the checkpoints. */
-        wts_verify_mirrors(conn, ckpt_vrfy_name, NULL);
+        /* FIXME-WT-15357 Checkpoint cursors are not compatible with disagg for now. */
+        if (!g.disagg_storage_config)
```

Comment:
@mariammojid I’m aware that verify doesn’t work in follower mode. However, I’d like to run mirrored tables on the follower, which requires verify to be enabled. Is there an existing ticket for adding verify support in follower mode that I can link here as `FIXME`? Once verify is supported for follower ingest tables, we can remove this. 

FYI, This is just a draft, I’ll add a better conditional later.